pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1567,7 +1547,13 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 		),
 		compute.TPMDomainConfigurator{},
 		compute.VSOCKDomainConfigurator{},
-		compute.NewLaunchSecurityDomainConfigurator(c.Architecture.GetArchitecture()),
+		compute.NewLaunchSecurityDomainConfigurator(architecture),
+		compute.NewRNGDomainConfigurator(
+			compute.RNGWithArchitecture(architecture),
+			compute.RNGWithUseVirtioTransitional(c.UseVirtioTransitional),
+			compute.RNGWithUseLaunchSecuritySEV(c.UseLaunchSecuritySEV),
+			compute.RNGWithUseLaunchSecurityPV(c.UseLaunchSecurityPV),
+		),
```

Comment:
```suggestion
		compute.NewRNGDomainConfigurator{
		Architecture: architecture,
	    UseVirtioTransitional: c.UseVirtioTransitional, 
	    UseLaunchSecuritySEV: c.UseLaunchSecuritySEV,
	    UseLaunchSecurityPV: c.UseLaunchSecurityPV,
		}
```

why not initialize struct like this? 

can you explain the benefit of adding an option?